data-prepper-plugins/parquet-codecs/src/main/java/org/opensearch/dataprepper/plugins/codec/parquet/PrimitiveDataTypeChecker.java

```diff
@@ -0,0 +1,40 @@
+package org.opensearch.dataprepper.plugins.codec.parquet;
+
+import org.apache.parquet.example.data.simple.SimpleGroup;
+import org.apache.parquet.schema.PrimitiveType;
+import org.apache.parquet.schema.Type;
+
+public class PrimitiveDataTypeChecker {
+
+    private PrimitiveDataTypeChecker(){
+
+    }
+
+    public static Object checkPrimitiveDataType(Type field, SimpleGroup simpleGroup, int fieldIndex) {
```

Comment:
This class should have a unit test. This is should be fairly straightforward because it provide a true function.

Reply:
This class is already being unit tested. It is just an extension of the functionality of the main codec class and is being unit tested when the codec is parsing the different data types since in the test class, we are comparing the resultant parsed value with the initial input provided.